### PR TITLE
UI ux improvements

### PR DESCRIPTION
**Pull request by jeanawhou**
@cameronsutter @Quiescent these are easy fixes for the ui-ux improvements 😅 I'm still working on the remaining tickets

Ticket:
https://airtable.com/appqY1Wa4HybeLzHs/tblTBZjhv1VH6kFPH/viw5M5GJxLKIzJq9j/recxMxL9iPXaQYWD3?blocks=hide

Update:
image

Ticket: https://airtable.com/appqY1Wa4HybeLzHs/tblTBZjhv1VH6kFPH/viw5M5GJxLKIzJq9j/rece5sUMXWIBKvv3x?blocks=hide

Update:
image

Ticket: https://airtable.com/appqY1Wa4HybeLzHs/tblTBZjhv1VH6kFPH/viw5M5GJxLKIzJq9j/recCvTwvSar17NUV6?blocks=hide

Update: 
image

TIcket: https://airtable.com/appqY1Wa4HybeLzHs/tblTBZjhv1VH6kFPH/viw5M5GJxLKIzJq9j/recZUCRqaB7OV6BV2?blocks=hide

Update: 
image

Ticket: https://airtable.com/appqY1Wa4HybeLzHs/tblTBZjhv1VH6kFPH/viw5M5GJxLKIzJq9j/recJIYS7jKsfhQNPV?blocks=hide

Update: 
image

Ticket: https://airtable.com/appqY1Wa4HybeLzHs/tblTBZjhv1VH6kFPH/viw5M5GJxLKIzJq9j/recTXeW42qfDYvOGq?blocks=hide

Update: 
image

Ticket: https://airtable.com/appqY1Wa4HybeLzHs/tblTBZjhv1VH6kFPH/viw5M5GJxLKIzJq9j/rec6OQT3J7bO11I3X?blocks=hide

Update: 
image

Tickets:
https://airtable.com/appqY1Wa4HybeLzHs/tblTBZjhv1VH6kFPH/viw5M5GJxLKIzJq9j/reccIw7gIEqAX1yPk?blocks=hide
AND
https://airtable.com/appqY1Wa4HybeLzHs/tblTBZjhv1VH6kFPH/viw5M5GJxLKIzJq9j/reciWNzhlrpoqoMfn?blocks=hide

Update: 
image

Ticket:
https://airtable.com/appqY1Wa4HybeLzHs/tblTBZjhv1VH6kFPH/viw5M5GJxLKIzJq9j/recMtg5sBbIMDhB2O?blocks=hide
AND
https://airtable.com/appqY1Wa4HybeLzHs/tblTBZjhv1VH6kFPH/viw5M5GJxLKIzJq9j/reckhjvppDmrhl9DV?blocks=hide

Update:
image

Ticket: https://airtable.com/appqY1Wa4HybeLzHs/tblTBZjhv1VH6kFPH/viw5M5GJxLKIzJq9j/rec5KCpK11m1hDQBp?blocks=hide

Update: 
image

Ticket:
https://airtable.com/appqY1Wa4HybeLzHs/tblTBZjhv1VH6kFPH/viw5M5GJxLKIzJq9j/rec86yfcGKRTS9kpa?blocks=hide
AND
https://airtable.com/appqY1Wa4HybeLzHs/tblTBZjhv1VH6kFPH/viw5M5GJxLKIzJq9j/recr1drO8ZowopUeA?blocks=hide

Update: 
image

Ticket: https://airtable.com/appqY1Wa4HybeLzHs/tblTBZjhv1VH6kFPH/viw5M5GJxLKIzJq9j/recVznPtl2L9rvsc9?blocks=hide

Update:

https://user-images.githubusercontent.com/19387007/204175362-a27db003-73ed-4695-bef2-5561b030f495.mp4

Ticket: https://airtable.com/appqY1Wa4HybeLzHs/tblTBZjhv1VH6kFPH/viw5M5GJxLKIzJq9j/recn5Ysf1afQX9G1R?blocks=hide

Update: 
image

### CODE REVIEW

**Comment by Quiescent** [comment]
@jeanawhou re. https://airtable.com/appqY1Wa4HybeLzHs/tblTBZjhv1VH6kFPH/viw5M5GJxLKIzJq9j/rec6OQT3J7bO11I3X?blocks=hide.
Perhaps I misunderstand this one, but it looks like the requirement is to make the table as wide as the buttons above it(?)
In the screenshot, the table is still wider.

**Comment by Quiescent** [comment]
@jeanawhou, I think you missed part two of this ticket: https://airtable.com/appqY1Wa4HybeLzHs/tblTBZjhv1VH6kFPH/viw5M5GJxLKIzJq9j/recMtg5sBbIMDhB2O?blocks=hide
I think you need to move "Give Feedback" from the blue links just above those buttons into a button like the other help buttons.

**Comment by Quiescent** [comment]
@jeanawhou , this one (https://airtable.com/appqY1Wa4HybeLzHs/tblTBZjhv1VH6kFPH/viw5M5GJxLKIzJq9j/rec5KCpK11m1hDQBp?blocks=hide) can be interpreted in so many ways.
I'm going to let @cameronsutter know on the ticket that it's waaay too confusing so that he can weigh in on what was actually wanted here.

**Comment by Quiescent** [comment]
Good job so far.  Let's ensure that the work that was done matches Cameron's expectations before merging this.

**Comment by jeanawhou** [comment]
> Good job so far. Let's ensure that the work that was done matches Cameron's expectations before merging this.

Thank you for checking @Quiescent 🙇

**Comment by jeanawhou** [comment]
@cameronsutter @Quiescent this is the updated width of the recent files table for this ticket https://airtable.com/appqY1Wa4HybeLzHs/tblTBZjhv1VH6kFPH/viw5M5GJxLKIzJq9j/rec6OQT3J7bO11I3X?blocks=hide

image

**Comment by cameronsutter** [comment]
Looks perfect, @jeanawhou. Were the other things fixed as well? If so, let's merge

File: main/modules/menus/file.js [hunk 1]
```diff
@@ -66,18 +66,6 @@ function buildFileMenu(fileURL, getTrialInfo) {
             }
           },
         },
-        {
```
Comment (Quiescent) [inline main/modules/menus/file.js]: This looks like a mistake.
We shouldn't remove this if there's no alternative for saving a project with a new name via the file menu.
@cameronsutter, can you please comment on this?

Comment (jeanawhou) [reply main/modules/menus/file.js]: @Quiescent reverted back 😅

**Comment by jeanawhou** [comment]
> Looks perfect, @jeanawhou. Were the other things fixed as well? If so, let's merge

@cameronsutter yes 😅

File: main/modules/menus/file.js (hunk 1 above)
Comment (cameronsutter) [reply main/modules/menus/file.js]: it looks like that was incorrect, but it seems to be reverted, so 👍

**Comment by Quiescent** [comment]
@jeanawhou, @cameronsutter asked that your changes be included in our last release of the year.
I rebased them on top of the release branch and removed all of my changes that you pulled in when you based your branch on `staging`.
I had to revert a change in `src/app/index.js` re. duplicating a file.
Please can you double-check that everything made it across?
(I haven't checked the content yet -I'm running out of time in the day- so it's entirely possible that I messed something up.)

**Pull request merged**